apps/web/src/pages/TalentNominations/NominationGroup/components/FullCareerExperiences.tsx

```diff
@@ -0,0 +1,251 @@
+import { useState } from "react";
+import { useIntl } from "react-intl";
+import NewspaperIcon from "@heroicons/react/24/outline/NewspaperIcon";
+
+import { Button, Heading, Well } from "@gc-digital-talent/ui";
+import { AwardExperience, Experience } from "@gc-digital-talent/graphql";
+
+import experienceMessages from "~/messages/experienceMessages";
+import useControlledCollapsibleGroup from "~/hooks/useControlledCollapsibleGroup";
+import {
+  compareByDate,
+  isAwardExperience,
+  isCommunityExperience,
+  isEducationExperience,
+  isPersonalExperience,
+  isWorkExperience,
+} from "~/utils/experienceUtils";
+
+import ExperienceByTypeAccordion from "./ExperienceByTypeAccordion";
+
+interface FullCareerExperiencesProps {
+  experiences?: Omit<Experience, "user">[];
```

Comment:
We've been moving away from using entire models as props for a component.  Could you declare a fragment here and use that instead?

Reply:
Oh, this is impossible because of the type and the experience card needs?

Reply:
yeah I wasn't able to do that in here

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/issues/12877